gradle.properties

```diff
@@ -26,11 +26,11 @@ developmentEnvironmentUserName = "Developer"
 # The string's content will be replaced with your mods version when compiled. You should use this to specify your mod's
 # version in @Mod([...], version = VERSION, [...])
 # Leave these properties empty to skip individual token replacements
-replaceGradleTokenInFile = Lib.java
-gradleTokenModId =
-gradleTokenModName =
+replaceGradleTokenInFile = Tags.java
+gradleTokenModId = GRADLETOKEN_MODID
+gradleTokenModName = GRADLETOKEN_MODNAME
```

Comment:
Why use the gradle tokens other than version? They were not used from the beginning for a reason

Reply:
The end result is the same (it's defined right above in that file) and this lets Tags.java match and correctly generate the `@Mod` line so that FML stops complaining that it can't find the version and has to assume it.

It's unlikely we'd rename a mod but I believe the idea is that if we did it would be one location.

```
[17:22:12] [Client thread/WARN] [betterloadingscreen]: Mod betterloadingscreen is missing the required element 'version' and a version.properties file could not be found. Falling back to metadata version 1.3.30-GTNH

Reply:
Thats not a problem of using a class with a different name though or using gradletokens. The problems *litteraly* is what the error says: the version is missing from the `@Mod` annotation. Thus, the easiest and most sensible solution would be to set this element (`@Mod(version = Lib.Mod.VERSION, ...)`). It already is using a gradletoken there. The rest is just overengineering for the sake of it

Reply:
Fair enough, I can revert that